src/main/java/com/titanic/airbnbclone/service/AccommodationService.java

```diff
@@ -86,13 +87,47 @@ public DeleteReservationResponseDto delete(Long accommodationId, Long reservatio
 
             return DeleteReservationResponseDto.builder()
                     .status(StatusEnum.SUCCESS.getStatusCode())
-                    .message(ReservationEnum.RESERVATION_CANCEL_SUCCESS.getMessage())
+                    .message(ReservationMessage.RESERVATION_CANCEL_SUCCESS.getMessage())
                     .build();
 
         } catch (Exception e) {
             return DeleteReservationResponseDto.builder()
                     .status(StatusEnum.ACCEPTED.getStatusCode())
-                    .message(ReservationEnum.RESERVATION_CANCEL_FAIL.getMessage())
+                    .message(ReservationMessage.RESERVATION_CANCEL_FAIL.getMessage())
+                    .build();
+        }
+    }
+
+    public ReservationInfoResponseDtoList getReservationInfo(HttpServletRequest request) {
+
+        try {
+            String userEmail = (String) request.getAttribute(OauthEnum.USER_EMAIL.getValue());
```

Comment:
조회하는 메서드같은경우 `@Transactional(readOnly = true)` 로 많이 하더라구요. 김영한님 같은경우 같은 서비스 클래스안에 조회가 많은지 아니면 CUD가 많은지 판단하고 클래스에 `@Transactional(readOnly =true)`나 아니면 잭처럼 그대로 넣고 필요할 때마다 Transactional을 다시 재정의 합니다. 조회만 하는 메서드같은경우에는 readOnly를 붙여주는걸 추천합니다.

Reply:
그리고 request에서 받아 email을 이용하여 account를 찾는 로직이 많습니다. 따로 private로 두는건 어떨까요?
```suggestion

    private Account getAccount(HttpServletRequest request) {
       String userEmail = (String) request.getAttribute(OauthEnum.USER_EMAIL.getValue());
       return accountRepository.findByEmail(userEmail);
    }
```

Reply:
1. 맞습니다. 영한님 조언대로 조회가 많은 서비스라면 `@Transactional(readOnly = true)` 어노테이션을 붙이는 게 좋은데 원래 그렇게 사용했는데 실수가 발생할 수 있어서 제거했습니다. 
실제로 엔티티를 삭제하는 메서드인데 서비스 위에 `@Transactional(readOnly = true)`가 붙어있어서 디버깅하는데 시간이 걸렸네요. 

써니 말씀대로 이 서비스가 쿼리 조회가 많다고 느끼고 그걸 내가 인지하고 있으면 사용해도 좋을 것 같습니다.

2. 주신 피드백 좋습니다!